app/_how-tos/azure-saml-sso-insomnia.md

```diff
@@ -56,17 +57,36 @@ Now that you have the single-sign on URL from Insomnia, you can create a new Mic
 1. [Navigate to the SSO settings](https://learn.microsoft.com/entra/identity/saas-apps/saml-toolkit-tutorial#configure-microsoft-entra-sso) for the Microsoft Entra SAML toolkit you just created.
 1. Configure the following SAML SSO settings:
    
-   | Azure setting | Value |
-   |--------------|-------|
-   | Identifier (Entity ID) | The **Audience Restriction** field in the Insomnia SSO settings. |
-   | Reply URL | The **SSO URL** in the Insomnia SSO settings. |
-   | Sign on URL | The **SSO URL** in the Insomnia SSO settings. |
+{% table %}
+columns:
+  - title: Azure setting
+    key: setting
+  - title: Value
+    key: value
+rows:
+  - setting: "Identifier (Entity ID)"
+    value: "The **Audience Restriction** field in the Insomnia SSO settings."
+  - setting: "Reply URL"
+    value: "The **SSO URL** in the Insomnia SSO settings."
+  - setting: "Sign on URL"
+    value: "The **SSO URL** in the Insomnia SSO settings."
+{% endtable %}
```

Comment:
```suggestion
{% capture table1 %}
{% table %}
columns:
  - title: Azure setting
    key: setting
  - title: Value
    key: value
rows:
  - setting: "Identifier (Entity ID)"
    value: "The **Audience Restriction** field in the Insomnia SSO settings."
  - setting: "Reply URL"
    value: "The **SSO URL** in the Insomnia SSO settings."
  - setting: "Sign on URL"
    value: "The **SSO URL** in the Insomnia SSO settings."
{% endtable %}
{% endcapture %}

{{ table1 | indent:3 }}
```
The numbering is broken here because the table isn't indented.